src/backend/InvenTree/order/models.py

```diff
@@ -523,6 +523,37 @@ def company(self):
         help_text=_('Date order was completed'),
     )
 
+    approved_by = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        verbose_name=_('Approved by'),
+    )
+
+    approved_date = models.DateField(
+        blank=True,
+        null=True,
+        verbose_name=_('Approve Date'),
+        help_text=_('Date order was approved'),
+    )
+
+    reject_reason = models.CharField(
+        max_length=128,
```

Comment:
128 chars is short and very specific for freeform text - is there a technical reason?

Reply:
Nope. Chosen based on other values for strings within the codebase. Happy to change it if it's short.

Reply:
All fields that use 128 chars are md5 checksums. Our default for short free text fields is 250 chars

Reply:
Aight. Will fix 😊